html/rendering/non-replaced-elements/the-fieldset-element-0/legend-float.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<title>legend and float</title>
```

Comment:
"floated legend" would be a better title?

Reply:
Maybe, but the title doesn't matter much and I think it's clear enough as-is. :)

Reply:
Actually, changed to "floated legend should not disappear" now.